src/google/protobuf/util/json_format.proto

```diff
@@ -114,3 +115,11 @@ enum EnumValue {
 message TestDefaultEnumValue {
   optional EnumValue enum_value = 1 [default = DEFAULT];
 }
+
+message TestEnumMap {
```

Comment:
Failure from CI:
- https://github.com/protocolbuffers/protobuf/actions/runs/7993753108/job/21830279412?pr=15887#step:13:3493
- https://github.com/protocolbuffers/protobuf/actions/runs/7993753108/job/21830264580?pr=15887#step:3:1046

Reply:
Fixed by renaming this message.